libs/ui/src/components/small-info-card/SmallInfoCard.tsx

```diff
@@ -0,0 +1,23 @@
+import Typography from '@mui/material/Typography';
```

Comment:
rename the parent folder from `small-info-card` to `card`, this folder will contain all types of card we could create in the future